streaming/rdm_multi_recv.c

```diff
@@ -85,7 +85,7 @@ static int repost_recv(int iteration) {
 int wait_for_recv_completion(int num_completions)
 {
 	int i, ret;
-	struct fi_cq_data_entry comp;
```

Comment:
The test sets
```
cq_attr.format = FI_CQ_FORMAT_DATA;
```
which would make fi_cq_data_entry correct here.  Is the problem in the test or actually in the verbs provider?

Tagging @a-ilango  as this may be a provider issue.